stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/options/message/MessageOptionsDialogFragment.kt

```diff
@@ -251,7 +244,7 @@ public class MessageOptionsDialogFragment : FullScreenDialogFragment() {
         messageListItemViewHolderFactory.withDecoratorProvider(messageOptionsDecoratorProvider) {
             viewHolder = it.createViewHolder(
                 binding.messageContainer,
-                MessageListItemViewTypeMapper.getViewTypeValue(messageItem, attachmentFactoryManager),
+                it.getItemViewType(messageItem),
```

Comment:
Here we're using `MessageListItemViewHolderFactory.getItemViewType` instead of `MessageListItemViewTypeMapper.getViewTypeValue`